README.rst

```diff
@@ -29,26 +29,24 @@ Use at the command line:
 
 1. In your current working directory type ``source /cds/group/pcds/pyps/conda/venvs/lcls_naming_tool/bin/activate``
 
-2. To check if a PV or device name is valid pipe in the name to the tool. For example, ``$ echo "MR2K4:KBO:PIP:01:PUMPSIZE" | ./lcls_naming_tool.py``
+2. To check if a PV or device name is valid pipe in the name to the tool. For example, ``$ echo 'MR2K4:KBO:PIP:01:PUMPSIZE' | ./lcls_naming_tool.py``
 
 3. To check if a list of names is valid pipe in the file name to the tool. For example, ``$ cat pvlist.txt | ./lcls_naming_tool.py`` (names should be separated by newline characters.)
 
-4. To view the current version add ``-v`` or ``--version``. For example, ``$ echo "MR2K4:KBO:PIP:01:PUMPSIZE" | ./lcls_naming_tool.py --version``
+4. To view the current version add ``-v`` or ``--version``. For example, ``$ echo 'MR2K4:KBO:PIP:01:PUMPSIZE' | ./lcls_naming_tool.py --version``
 
 
 Use in a Python script:
 
-1. In your current working directory set up the environment: 
+1. In your current working directory set up the environment:
 
 ``source /cds/group/pcds/pyps/conda/venvs/lcls_naming_tool/bin/activate``
 
-``git clone git@github.com:pcdshub/lcls_naming_tool.git``
+``pip install lcls_naming_tool``
 
-``export PYTHONPATH=$PWD/lcls_naming_tool/lcls_naming_tool``
+2. Add ``from lcls_naming_tool import load_taxons, validate`` to the top of your file.
 
-2. To call the LCLS Naming Tool module ``from lcls_naming_tool import load_taxons, validate``
-
-5. In your ``main`` function ``load_taxons()`` should be called first. It takes no parameters and loads all the approved taxons in JSON format. The function ``validate()`` takes a PV or device name in string format as a parameter and returns ``True`` for a valid name or ``False`` for an invalid name.
+3. In your ``main`` function ``load_taxons()`` should be called first. It takes no parameters and loads all the approved taxons in JSON format. To check a PV or device name call the ``validate()`` function. ``validate()`` accepts a string as a parameter and returns ``True`` if the name is valid or  ``False`` if it's invalid.
```

Comment:
Nitpick: the docs here are strangely specific. I think this section needs some revisions, e.g. you don't want nor expect someone else to source your venv and make edits to it via pip install, do you?

Reply:
Definitely don't want users to modify the venv lol. Could I change permissions for the virtual environment at /cds/group/pcds/pyps/conda/venvs/lcls_naming_tool and remove the write permissions? Something like `chmod -R 555`?

Reply:
I think Zach's point is that these are instructions specific to our installation of the naming tool in our filesystem/conda environment.  Normally instructions include some boilerplate on how to setup a development environment, install the tool in that environment with the necessary dependencies, and start up the app (configuring and calling the necessary scripts)

Reply:
We can certainly lock down the existing venv, but I think the main critique was the mismatch in audience.

Reply:
Ah okay, I wasn't sure how the user will use the naming tool. I'll add some instructions for setting up the development environment, installing dependencies, and start up the app.